src/test/scala/com/workday/warp/persistence/mysql/WarpSlickDslSpec.scala

```diff
@@ -52,6 +60,57 @@ class WarpSlickDslSpec extends WarpJUnitSpec with CorePersistenceAware {
 
     val query2 = TestExecution.filter(_.endTime isWithinPast "5 MINUTE")
     this.persistenceUtils.runWithRetries(query2.result, 5).size shouldBe 3
+
+  }
+
+
+  @Test
+  @Category(Array(classOf[UnitTest]))
+  /** Tests YEAR dsl. */
+  def returnYear(): Unit = {
+    val testExecution: TestExecutionRowLike = this.persistenceUtils.createTestExecution(methodSignature1, new Date, 1.0, 10)
+    val timeStamp: Rep[Timestamp] = testExecution.startTime
```

Comment:
is `startTime` implicitly lifted into a `Rep`?

Reply:
I'm guessing since it's represented as a column in the table it becomes a rep?

Reply:
I believe slick implicitly lifts this into a `Rep`